Testscripts/Linux/VERIFY-BOOT-ERROR-WARNINGS.py

```diff
@@ -97,4 +99,16 @@ def RemoveIgnorableMessages(message_list, keywords_xml_node):
         return None
 
 
+def CollectLogs():
+    logfiles = ['/var/log/messages']
+    hostname = os.uname()[1]
+    for logfile in logfiles:
+        if os.path.exists(logfile):
+            dst = "{}-{}.txt".format(hostname, os.path.basename(logfile))
+            try:
+                copyfile(logfile, dst)
```

Comment:
It is good to log the file copying to the user. Please add log printing.

Reply:
Thanks @juhlee-microsoft ! I'll add log printing now.

Reply:
Updated. Now print these two lines:
```
05/11/2020 11:15:45 AM : INFO : Copying /var/log/messages to /home/azureuser/LISAv2-OneVM-walaautodev-UB34-20200510201003-role-0-messages.txt...
05/11/2020 11:15:45 AM : INFO : Copy all logs finished!
```

Reply:
@yuxisun1217 could you don't use basename? what about use  *'**-var-log-message.txt' for not confusing other distro? it's not common used for ubuntu, so only leave '*-message.txt' not useful to other distro.

Reply:
Apparently I merged it. Can you work on new PR please?

Reply:
@harz566 @juhlee-microsoft OK. I'll open a new PR to change the filename. Thanks!